go/vt/wrangler/materializer.go

```diff
@@ -1064,6 +1074,36 @@ func (mz *materializer) deploySchema(ctx context.Context) error {
 	})
 }
 
+func stripTableFKConstraints(ddl string) (string, error) {
+
+	ast, err := sqlparser.ParseStrictDDL(ddl)
+	if err != nil {
+		return "", err
+	}
+
+	stripFKConstraints := func(cursor *sqlparser.Cursor) bool {
+		switch node := cursor.Node().(type) {
+		case sqlparser.DDLStatement:
+			if node.GetTableSpec() != nil {
+				noFKConstraints := make([]*sqlparser.ConstraintDefinition, 0, len(node.GetTableSpec().Constraints))
```

Comment:
Since we are now filtering the slice, allocating extra capacity up front with the length of the input slice could be wasteful. I'd probably do `var noFKConstraints []*sqlparser.ConstraintDefinition` and let `append()` do its thing.